### DISCUSSION TITLE
x-trap and entangle issue

### DISCUSSION THREAD

**Discussion by delta1186** [alpinejs/alpine#2130] [7. Bugs (archived)]
Hey @calebporzio I am having is when using x-trap on a component where I am using @entangle in x-data I cannot tab inputs.  

Here is a phpsandbox demonstrating the issue.  

https://phpsandbox.io/e/x/qwio1?layout=EditorPreview&defaultPath=%2F&theme=dark&showExplorer=yes&openedFiles=  

I am running the latest tags of alpine, trap, and livewire.  This was just working a few days ago.  I am wondering if something changed after the focus-trap #2072 was merged.  In fact I just tested the following.

If I remove the x.x in the cdn links and use 3.3.5 the example above works when using entangle.  So something does appear to be up the focus-trap commit.
```
<!-- Alpine Plugins -->
<script defer src="https://unpkg.com/@alpinejs/trap@3.x.x/dist/cdn.min.js"></script>
 
<!-- Alpine Core -->
<script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
```

**Comment by emaia** [comment]
Same problem here. Starting with v3.4.0 x-trap doesn't work with entangle.

Error in console: 
```Uncaught Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times```
